searchcore/src/tests/proton/documentdb/lid_space_compaction/lid_space_compaction_test.cpp

```diff
@@ -513,11 +497,8 @@ TEST_F(JobTest, job_can_restart_documents_scan_if_lid_bloat_is_still_to_large)
     // We simulate that the set of used docs have changed between these 2 runs
     EXPECT_FALSE(run()); // move 9 -> 2
     endScan();
-    assertJobContext(2, 9, 1, 0, 0);
-    EXPECT_EQ(2u, _handler->_iteratorCnt);
-    EXPECT_FALSE(run()); // does not find 8 in first scan
-    EXPECT_FALSE(run()); // move 8 -> 3
```

Comment:
Does this test still test what it should, or is it perhaps not necessary anymore/needs a rewrite.

Reply:
I believe this is testing the shouldRestartScanDocuments() handling in LidSpaceCompactionJob::scanDocuments(). I don't think we need to change anything in this test.

Reply:
It did set maxdocs to 3. The mayhem of multiple parameters with similar type and default value...
Fixed in https://github.com/vespa-engine/vespa/pull/16142